open-api/rest-catalog-open-api.yaml

```
@@ -3789,6 +3986,21 @@ components:
             EmptyResponse:
               $ref: '#/components/examples/ListNamespacesEmptyExample'
 
+    PayloadTooLargeResponse:
+      description:
+        Payload is too large.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/IcebergErrorResponse'
+          example: {
+            "error": {
+              "message": " Payload is too large for PlanTable, please call PreplanTable to distribute table scan planning",
```

We try to be a bit more straightforward and clear in error messages by removing identifiers that the caller doesn't understand (i.e. `PreplanTable`) and being direct (removing "please" or personal pronouns like "I" and "we").

```suggestion
              "message": "Cannot plan scan: too many task results; use /preplan to split planning across multiple requests",
```

This also raises a question: what if you've sent a plan task? I think that we may want to state in the description of the plan route that if there is a valid plan task from the service, it must attempt to send all result tasks.

---

will fix this to be direct and also amend plan table description.